components/ProgressView/examples/ProgressViewPillExample.m

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialContainerScheme.h"
+#import "MaterialPalettes.h"
+#import "MaterialProgressView.h"
+
+static const CGFloat kProgressViewHeight = 4;
+
+@interface ProgressViewPillExample : UIViewController
+
+@property(nonatomic, strong) UIView *container;
+
+@property(nonatomic, strong) MDCProgressView *progressView;
+@property(nonatomic, strong) UILabel *progressLabel;
+
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
+
+@end
+
+@implementation ProgressViewPillExample
+
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
+  self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
+  if (self) {
+    self.title = @"Progress View Pill";
+    self.navigationItem.rightBarButtonItem =
+        [[UIBarButtonItem alloc] initWithTitle:@"Advance"
+                                         style:UIBarButtonItemStylePlain
+                                        target:self
+                                        action:@selector(didPressAdvanceButton:)];
+    self.navigationItem.rightBarButtonItem.accessibilityIdentifier = @"advance_button";
+
+    self.containerScheme = [[MDCContainerScheme alloc] init];
+  }
+  return self;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.view.backgroundColor = self.containerScheme.colorScheme.backgroundColor;
+
+  self.container = [[UIView alloc] initWithFrame:self.view.bounds];
+  [self.view addSubview:self.container];
```

Comment:
why do we need a container and not just use the viewcontroller's view?

Reply:
Makes it easier to inset the content via `layoutMargins` later on.